docs/src/main/paradox/handling-blocking-operations-in-pekko-http-routes.md

```diff
@@ -63,8 +65,8 @@ provided dispatcher would be used. Both these dispatchers are ForkJoin pools by
 not best suited for blocking operations.
 @@@
 For example, the above screenshot shows an Apache Pekko FJP dispatchers threads,
-named "`default-akka.default-dispatcher2,3,4`" going into the blocking state, after having been idle. 
-It can be observed that the number of new threads increases, "`default-akka.actor.default-dispatcher 18,19,20,...`" 
+named "`m-pekko.actor.default-dispatcher-6,7`" going into the blocking state, after having been idle. 
```

Comment:
why 'm-pekko'? - there is also an 'default-akka' on the next like - should these not both become 'default-pekko'?

Reply:
Good catch on the mismatched names. See other comment about why `m-pekko` originally: I was using the typed actor system. Will fix shortly.